### DISCUSSION TITLE
Can we customise axis (not just value or label) ?

### DISCUSSION THREAD

**Discussion by vivekgupta4Git** [patrykandpatrick/vico#763] [Questions] [answered]
### Question

<img width="453" alt="image" src="https://github.com/patrykandpatrick/vico/assets/91813403/e52baa22-44d1-4da7-a157-cb9fa1d0b996">

Our designer is giving us very hard time as we have integrated this amazing library in our app and their expectation is very high from the library.  As you can see in the pic, graph's y axis include a Giraffe, is it possible to draw a graph like this? 
I have tried using decoration api to decorate the drawable behind the graph but all in vain. 

### Vico version(s)

2.0.0-alpha.11

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello!
>Can we customise axis (not just value or label) ?

Yes. There are many other customization options. 

>is it possible to draw a graph like this?

Yes. You can create a `VerticalAxis` subclass and draw whatever you want. You can use `Insets` API to reserve enough space.

**Reply by vivekgupta4Git** [reply]
is there any sample available for custom axis implementation as I couldn't find one?

**Reply by Gowsky** [reply]
Yes, the default implementation is an example of a `VerticalAxis` implementation. You seem to be asking a generic question and seeking a specific answer.

**Reply by vivekgupta4Git** [reply]
Well, I have managed to create the desired output but with a tweak thus was seeking one more example.

**Reply by Gowsky** [reply]
Let us know if you have any specific questions.
